aicspylibczi/CziFile.py

```diff
@@ -158,31 +173,59 @@ def scene_bounding_box(self, index: int = -1):
             (x0, y0, w, h) for the specified scene
 
         """
-        bbox = self.reader.read_scene_wh(index)
-        ans = (bbox.x, bbox.y, bbox.w, bbox.h)
-        return ans
+        bbox = self.reader.read_scene_bounding_box(index)
+        return bbox
 
-    def scene_height_by_width(self, index: int = -1):
+    def get_all_tile_bounding_boxes(self, **kwargs):
         """
-        Get the size of the scene (Y, X) / (height, width) for the specified Scene. The default is to return
-        the size of the first Scene but Zeiss allows scenes to be different sizes thought it is unlikely to encounter.
-        This module will warn you on instantiation if the scenes have inconsistent width and height.
 
-        Parameters
-        ----------
-        index
-            specifies the index of the Scene to get the height and width of
+        :param kwargs:
+        :return:
+        """
+        plane_constraints = self.czilib.DimCoord()
+        [
+            plane_constraints.set_dim(k, v)
+            for (k, v) in kwargs.items()
+            if k in CziFile.ZISRAW_DIMS
+        ]
+        m_index = self._get_m_index_from_kwargs(kwargs)
```

Comment:
Nit: this is the same as above from `get_tile_bouding_box`, may want to make a `_` function to do this operation in a single place so that the two don't get out of sync if you ever need to update them / find a bug.